doc/contributing/documentation_guide.md

```diff
@@ -199,6 +199,36 @@ will be autolinked:
 If not, or if you suppress autolinking, consider forcing
 [monofont](rdoc-ref:RDoc::MarkupReference@Monofont).
 
+### Explicit Links
+
+When writing an explicit link, follow these guidelines.
+
+#### +rdoc-ref+ Scheme
+
+Prefer the +rdoc-ref+ scheme for:
+
+- A link in core documentation to other core documentation.
+- A link in core documentation to documentation in a standard library package.
```

Comment:
My feelings have not changed.  We should not recommend an approach unless we can figure out how to make it work ourselves.

One issue with using rdoc-ref in this case is if you build just the core documentation without the standard library documentation, the rdoc-ref link will not work.  I don't think that affects docs.ruby-lang.org.  It looks like for recent releases, ruby-doc.org also builds them both together.  So I don't think we should avoid this recommendation, but we must figure out how to get it to work first.  I'll see if I can determine how to get it to work.

Reply:
I'll work on it, too.

Reply:
Got it.  I'll push soon.

Reply:
Fixed.